src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1134,6 +1137,17 @@ public static void setOverrideLoadConfig(Supplier<Config> loadConfig)
         exception
     }
 
+    public enum CompressorType
+    {
+        lz4,
+        none,
+        noop,
+        snappy,
+        deflate,
+        zstd,
+        custom,
+    }
```

Comment:
can we move this into SSTableCompressionOptions ?

Reply:
yes, I agree, I was planning to do this on merge.